app/lib/proton/index.js

```diff
@@ -36,10 +38,33 @@ class protonRunner {
 
     const args = ['--url', url]
 
-    return this.__runCargoCommand(
-      ['run', '--features', 'dev'],
-      args
-    )
+    const startDevProton = () => {
+      return this.__runCargoCommand(
+        ['run', '--features', 'dev'],
+        args
+      )
+    }
+
+    // Start watching for proton app changes
+    chokidar
```

Comment:
Should stop the watcher on protonRunner's stop() method being called.

Reply:
Done here #e85aaba52bc5cff3f12903374faacefa916f290c